pkg/peer/dcasconfig.go

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
for a followup: a markdown document describing the cas and ipfs features, how it works, configuration, and usage would be helpful.